knative-operator/pkg/controller/knativeserving/kourier/kourier.go

```diff
@@ -17,13 +17,13 @@ import (
 )
 
 var (
-	log  = common.Log.WithName("kourier")
-	path = "deploy/resources/kourier/kourier-latest.yaml"
+	log             = common.Log.WithName("kourier")
+	defaultManifest = "deploy/resources/kourier/kourier-latest.yaml"
```

Comment:
If you go by the suggestion below, we can just use `os.Getenv("KOURIER_MANIFEST_PATH")` here, I think?